fcd_weight_scale_mrp/wizards/fcd_weight_scale_mrp.py

```diff
@@ -18,16 +18,17 @@ def remove_last_weight_scale(self):
             # Create mrp.unbuild for mrp.production
             if log_id.production_id:
                 production_id = log_id.production_id
-                unbuild_id = self.env['mrp.unbuild'].create({
-                    'product_id': production_id.product_id.id,
-                    'product_uom_id': production_id.product_uom_id.id,
-                    'product_qty': production_id.product_qty,
+
+                unbuild = self.env['mrp.unbuild'].new({
                     'lot_id': production_id.lot_producing_id.id,
                     'mo_id': production_id.id,
-                    'company_id': production_id.company_id.id,
-                    'location_id': production_id.location_dest_id.id,
+                    'location_id': production_id.location_src_id.id,
```

Comment:
Pending to be fixed when production is fixed as well:
```suggestion
                    'location_id': production_id.location_dest_id.id,
```